models.py

```diff
@@ -1,3 +1,7 @@
+import pandas as pd
+from statsmodels.tsa.arima_model import ARIMA
```

Comment:
This could cause confusion between ARIMA and arima. Currently if you import `models` you could call `models.ARIMA` and it would call this by mistake. I would do "import package" rather than "from package import arima"